Lib/test/test_gettext.py

```diff
@@ -461,116 +462,174 @@ def setUp(self):
         GettextBaseTest.setUp(self)
         self.mofile = MOFILE
 
+    @contextlib.contextmanager
```

Comment:
This seems like it might be more generally useful.  Maybe move it to `test.support`?

Reply:
Not sure. It tests strictly the particular form of the deprecation message.